rpc/server.py

```diff
@@ -620,6 +620,11 @@ def ping() -> dict:
     return {"status": "OK"}
 
 
+@app.route('/github-actions/ping')
```

Comment:
@sevenearths why don't you use the existing ping endpoint?

Reply:
There was a reason, but I can't remember what it it. Have switched it over to using the ping end point

Reply:
I'll do it as a jsonrpc call. Np